src/bin/odata-compiler.ts

```diff
@@ -0,0 +1,125 @@
+process.env.PINEJS_CACHE_FILE =
+	process.env.PINEJS_CACHE_FILE || __dirname + '/.pinejs-cache.json';
+
+import * as _abstractSql from '../sbvr-api/abstract-sql';
+import * as _sbvrUtils from '../sbvr-api/sbvr-utils';
+import * as _uriParser from '../sbvr-api/uri-parser';
+
+import * as program from 'commander';
+import * as fs from 'fs';
+import '../server-glue/sbvr-loader';
+
+// tslint:disable:no-var-requires
+const { version } = JSON.parse(
+	fs.readFileSync(require.resolve('../../package.json'), 'utf8'),
```

Comment:
It would be nice if we also bumped the `engines` field in package.json to `"node": ">=8.0.0",`  (even thought that's not the only node 8  fn that we are using).

Reply:
I don't think anything on this line requires nodejs 8?  That said the package.json engines is really really out of date so I've updated it to the current requirements (and added a note to do a matching change in https://github.com/balena-io/pinejs/pull/264)